test/rest_api/test_storage_service.py

```diff
@@ -95,3 +95,40 @@ def test_storage_service_keyspace_offstrategy_compaction_tables(cql, this_dc, re
             assert resp.status_code == requests.codes.bad_request
 
     cql.execute(f"DROP KEYSPACE {keyspace}")
+
+def test_storage_service_keyspace_scrub(cql, this_dc, rest_api):
+    keyspace = new_keyspace(cql, this_dc)
```

Comment:
You should use here "with new_test_keyspace" just like you use new_test_table_below - so if the test fails., the keyspace gets deleted.

Reply:
right, will fix

Reply:
Fixed in v2.